subworkflows/local/reference_indexing.nf

```diff
@@ -125,17 +128,33 @@ workflow REFERENCE_INDEXING {
     ch_dbsnp = ch_dbsnp
         .filter { it[1] != "" }
 
+    // Elongate reference for circularmapper if requested
+    if ( params.mapping_tool == "circularmapper" ) {
+        // Throw errors if required parameters are missing
+        ch_elongated_for_gunzip = ch_elongated_reference
+                        .filter{ it[1] != "" || it[2] != "" }
```

Comment:
Use named elements rather than `it[]` I don't know what `it[1]` and `it[2]` are from this PR diff...